code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -200,6 +221,17 @@
 	use_power(power_usage)
 	update_parents()
 
+/obj/machinery/atmospherics/components/unary/thermomachine/attackby(obj/item/assembly/signaler/anomaly/pyro, mob/living/user, params)
+	if(pyro_anomaly_core || !allows_cooling)
+		balloon_alert(user, "already an anomaly core!")
+		return
+	else
+		balloon_alert(user, "inserted anomaly core")
```

Comment:
```suggestion
		balloon_alert(user, "inserted [pyro.name]")
```
Could do this for above as well

Reply:
Fixed

Reply:
I didn't make the above in case there is a possible cryo anomaly core in the future in which case it would need to be changed.